packages/editor-ui/src/components/RunData.vue

```diff
@@ -1591,6 +1595,20 @@ defineExpose({ enterEditMode });
 				</N8nText>
 			</div>
 
+			<div v-else-if="isTrimmedManualExecutionDataItem" :class="$style.center">
+				<N8nText bold color="text-dark" size="large">
+					{{ i18n.baseText('runData.trimmedData.title') }}
+				</N8nText>
+				<N8nText>
+					{{ i18n.baseText('runData.trimmedData.message') }}
+				</N8nText>
+				<N8nButton size="small">
+					<a :href="`/workflow/${workflowsStore.workflowId}/executions`">
```

Comment:
This seems to do a full navigation that reload the entire page. Would it be possible to use vue router link?

Reply:
Unfortunately the styling of `N8nLink` does not match what Giulio requested:

![Capture 2024-12-31 at 09 49 30@2x](https://github.com/user-attachments/assets/c006bb67-cbfe-44fd-af63-37847f612d09)

The button is too big and the color is not available in the `N8nLink` themes.

Reply:
In that case we can use the `N8nRoute` component directly and style it how we wish

Reply:
Thank you, please see [33fc9f8](https://github.com/n8n-io/n8n/pull/12342/commits/33fc9f8609e06335596acabf69bc18e98a5490b8)